### DISCUSSION TITLE
既然Modern.js 基于 Hono 框架 搭建了自己的 Web 服务器，那为什么还要在BFF中增加express和koa的运行时呢

### DISCUSSION THREAD

**Discussion by yunloon** [web-infra-dev/modern.js#6344] [General]
如题，在文档中Web服务器一篇中，提到了"[Modern.js 基于 Hono 框架 搭建了自己的 Web 服务器。Hono 是一个小型、简单且超快速的基于 Web 标准的框架，它能在任何 JavaScript 运行时上运行。](https://modernjs.dev/guides/concept/server.html)"

既然hono可以当做web服务器来使用，那么在使用bff的时候为什么要增加express和koa的运行时呢，直接使用内置的hono不可以吗？感觉增加express和koa的运行时是有点多余。

希望大佬解答

**Comment by zllkjc** [comment]
我们有计划这样做，预期会在明年第一个季度发布这个特性

**Comment by zllkjc** [comment]
这里之所以有 express / koa 的原因是在 modern.js 1.0 的时候，没有使用 hono，现在可以启动了

**Reply by yunloon** [reply]
辛苦 感谢解答

**Comment by zllkjc** [comment]
目前基于 Hono 的 BFF 已经盒马了，我们将在近期提供文档。
